### PR TITLE
Reword spread rules in `Ranking criteria/osu!mania`

### PR DESCRIPTION
**Pull request by osuAntalf**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

This proposal was made for clarification to other game modes as mania is able to rank X spreads. This was raised by other multiple game modes like taiko and catch. A consensus came to light once we found a better way to word the rule as to not create confusion between mods. This was discussed as well with the community and was given a pretty positive response overall, you can find all of it here https://osu.ppy.sh/community/forums/topics/1746229?n=1

In conclusion, this will make the rule more clear to everyone and will keep the integrity of how X spreads are ranked in mania as a whole.

### CODE REVIEW

**Comment by osuAntalf** [comment]
Discussed with me and the other 3 mania NATs as well as leaders. This is mostly to clarify a confusion regarding how the spread ruling was worded. Mostly catered to being consistent with the rest of the RC an avoid further confusion.

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
...
-  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a spread starting at least 2 difficulty levels below the highest difficulty.
-  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a spread starting at least 1 difficulty level below the highest difficulty.
+  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, OR each keymode must provide a proper spread containing at least 4 difficulties (including the highest difficulty).
+  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, OR each keymode must provide a proper spread containing at least 3 difficulties (including the highest difficulty).
+  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, OR each keymode must provide a proper spread containing at least 2 difficulties (including the highest difficulty).
```
Comment (aceticke) [inline wiki/Ranking_Criteria/osu!mania/en.md:67]:
```suggestion
  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, **OR **each keymode must provide a proper spread containing at least 4 difficulties (including the highest difficulty).
  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a proper spread containing at least 3 difficulties (including the highest difficulty).
  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a proper spread containing at least 2 difficulties (including the highest difficulty).
```
keeping the bold here is nice for clarity and helps to separate them as two options for drain time

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 2]
```diff
@@ -62,9 +62,9 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
...
 - **If the [drain time](/wiki/Beatmap/Drain_time) of each difficulty is...**
-  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, **OR** each keymode must provide a spread starting at least 3 difficulty levels below the highest difficulty.
-  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a spread starting at least 2 difficulty levels below the highest difficulty.
-  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a spread starting at least 1 difficulty level below the highest difficulty.
+  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, OR each keymode must provide a proper spread containing at least 4 difficulties (including the highest difficulty).
```
Comment (juankristal) [inline wiki/Ranking_Criteria/osu!mania/en.md:65]: Any reason why the bolded or got changed to regular or? I think bolded is better in this case.

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 1 above)
Comment (aceticke) [inline wiki/Ranking_Criteria/osu!mania/en.md:67]: what exactly is a "proper spread"? if we are using vague terms like this, they should be properly defined, however a footnote for this seems unnecessary, so is there a better phrasing of what you mean? perhaps could do:

```suggestion
  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, OR each keymode must provide a spread containing at least 4 difficulties (including the highest difficulty) without skipping any difficulty levels.
  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, OR each keymode must provide a spread containing at least 3 difficulties (including the highest difficulty) without skipping any difficulty levels.
  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, OR each keymode must provide a spread containing at least 2 difficulties (including the highest difficulty) without skipping any difficulty levels.
```

**Review by aceticke — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 1 above)
Comment (juankristal) [reply wiki/Ranking_Criteria/osu!mania/en.md:67]: The point of the clarification is exactly to avoid difficulty levels (given a level or a name are different terms in mania). A spread is defined equally between all modes but mania adds the specific distinction `On difficulties Insane and harder, a proper spread is defined by relative difficulty gaps similar to those formally defined between lower difficulties`

I think linking that into the `spread` word would be a nice addition though.

Comment (juankristal) [reply wiki/Ranking_Criteria/osu!mania/en.md:67]: Although now that I see it closely, its literally the next sentence so probably not needed.

Comment (Protastic101) [reply wiki/Ranking_Criteria/osu!mania/en.md:67]: The point of this PR was to get rid of the term "difficulty level" since that implies it must fall within ENHIX conventions. Currently, mania is allowed to have X only spreads under this rule, and by reintroducing "difficulty level" into the language, we come back to the confusion that technically, an X diff would need at minimum, an I diff to be rankable when that's not been the common interpretation within mania.

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 3]
```diff
@@ -62,9 +62,9 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
...
-  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a spread starting at least 2 difficulty levels below the highest difficulty.
-  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a spread starting at least 1 difficulty level below the highest difficulty.
+  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, **OR** each keymode must provide a proper spread containing at least 4 difficulties (including the highest difficulty).
+  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a proper spread containing at least 3 difficulties (including the highest difficulty).
+  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a proper spread containing at least 2 difficulties (including the highest difficulty).
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/osu!mania/en.md:67]: there should definitely be a footnote on "proper spread"

```suggestion
  - **...lower than 2:30**, the lowest difficulty of each included key mode cannot be harder than a Normal, **OR** each key mode must provide a proper spread containing at least 4 difficulties.
  - **...between 2:30 and 3:15**, the lowest difficulty of each included key mode cannot be harder than a Hard, **OR** each key mode must provide a proper spread containing at least 3 difficulties.
  - **...between 3:15 and 4:00**, the lowest difficulty of each included key mode cannot be harder than an Insane, **OR** each key mode must provide a proper spread containing at least 2 difficulties.
```

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 4]
```diff
@@ -62,9 +62,9 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
...
 - On difficulties Insane and harder, a proper spread is defined by relative difficulty gaps similar to those formally defined between lower difficulties.
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/osu!mania/en.md:68]:
```suggestion
- A "proper" spread is defined as a spread with gaps in difficulty similar to those between lower [difficulty levels](/wiki/Beatmap/Difficulty#difficulty-levels) as specified in the [difficulty-specific criteria](#difficulty-specific).
```

Comment (osuAntalf) [reply wiki/Ranking_Criteria/osu!mania/en.md:68]: I think this goes against the reason why we are making this PR in the first place as I mentioned in the first one. It's mostly to clear a confusion between other game modes since mania behaves differently, changing this will be going back to square one not allowing something that is allowed in mania.

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 3 above)
Comment (osuAntalf) [reply wiki/Ranking_Criteria/osu!mania/en.md:67]: Fixed "keymode" but kept "(including the highest difficulty)" since this is the whole purpose of this PR, to allow the use of X spreads and not confuse anyone from other game modes about this.

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 4 above)
Comment (Walavouchey) [reply wiki/Ranking_Criteria/osu!mania/en.md:68]: i'm not sure i follow. this suggestion intends to reword this line so it becomes easier to understand, without changing the meaning, and it has nothing to do with other game modes.

**Review by osuAntalf — COMMENTED** [review]
Added outdate tags (I hope)

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 4 above)
Comment (Walavouchey) [reply wiki/Ranking_Criteria/osu!mania/en.md:68]: edited as per [conversation on discord](https://discord.com/channels/188630481301012481/218677502141399041/1097298652017467535)

```suggestion
A "proper" spread *for difficulties Insane and harder* is defined as a spread with gaps in difficulty similar to those between lower [difficulty levels](/wiki/Beatmap/Difficulty#difficulty-levels) as specified in the [difficulty-specific criteria](#difficulty-specific).
```

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 5]
```diff
@@ -62,9 +62,9 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
...
-  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a spread starting at least 2 difficulty levels below the highest difficulty.
-  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a spread starting at least 1 difficulty level below the highest difficulty.
+  - **...lower than 2:30**, the lowest difficulty of each included key mode cannot be harder than a Normal, **OR** each keymode must provide a proper spread containing at least 4 difficulties (including the highest difficulty).
+  - **...between 2:30 and 3:15**, the lowest difficulty of each included key mode cannot be harder than a Hard, **OR** each keymode must provide a proper spread containing at least 3 difficulties (including the highest difficulty).
+  - **...between 3:15 and 4:00**, the lowest difficulty of each included key mode cannot be harder than an Insane, **OR** each keymode must provide a proper spread containing at least 2 difficulties (including the highest difficulty).
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/osu!mania/en.md:67]: this can be explained more concisely, as [explained on discord](https://discord.com/channels/188630481301012481/218677502141399041/1097310783576211516)

```suggestion
  - **...lower than 2:30**, the lowest difficulty of each included key mode cannot be harder than a Normal, **OR** each key mode must provide a proper spread containing at least 4 difficulties in total.
  - **...between 2:30 and 3:15**, the lowest difficulty of each included key mode cannot be harder than a Hard, **OR** each key mode must provide a proper spread containing at least 3 difficulties in total.
  - **...between 3:15 and 4:00**, the lowest difficulty of each included key mode cannot be harder than an Insane, **OR** each key mode must provide a proper spread containing at least 2 difficulties in total.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 5 above)
Comment (osuAntalf) [reply wiki/Ranking_Criteria/osu!mania/en.md]: This has been committed as per discussion on discord.
